philips_android_tv/media_player.py

```diff
@@ -355,12 +355,26 @@ def getChannel(self):
                         else:
```

Comment:
Could you remove this and add it to another PR?

Should be separate and I am not sure if every user is happy, if now only the favorite channels are displayed.
We should make this an option in settings.

This code is not perfect, but if it works, than that is already very much awesome and wow 😁